docs/paper/dev/guides/pdc.md

```diff
@@ -0,0 +1,132 @@
+---
+slug: /dev/pdc
+---
+
+# Persistent Data Container (PDC)
+
+The Persistent Data Container (PDC) is a way to store custom data on a whole range of objects; such as items, entities, and blocks.
+The Full list of classes that support the PDC are:
+
+- [Chunk](#chunk)
+- [World](#world)
+- [Entity](#entity)
+- [TileState](#tilestate)
+- [ItemMeta](#itemmeta)
+- [Structure](#structure)
+
+## What is it used for?
+In the past, developers resorted to storing data within the NBT tags of an object. 
+This was a very hacky way of storing data, and was prone to breaking if the NBT tags were changed. 
+The PDC is a much more stable way of storing data, and is much easier to use.
+
+As the name implies, the PDC is persistent. This means that the data stored in the PDC will be saved to disk, 
+and will be loaded back into the object when the server is restarted. This is great for storing attributes of an object,
+such as the owner of an item, or the custom name of an entity.
+
+## Adding Data
+To store data in the PDC, there are a few things you need first. The first is a `NamespacedKey` which is used to identify the data.
+The second is a `PersistentDataContainer` which is the object you want to store the data on. The third is the data itself.
+
+```java
+// Create a NamespacedKey
+NamespacedKey key = new NamespacedKey(pluginInstance, "example-key");
+
+ItemStack item = new ItemStack(Material.DIAMOND);
+// ItemMeta implements PersistentDataHolder so we can get the PDC from it
+ItemMeta meta = item.getItemMeta();
+item.getPersistentDataContainer().set(key, PersistentDataType.STRING, "I love Tacos!");
+item.setItemMeta(meta);
+```
+
+:::note NamespacedKeys
+
+It is considered good practice to store the `NamespacedKey`'s and reuse them.
```

Comment:
This can confuse people since we talk about a store system so maybe just say that you should reuse the key but not store (some people can thinks about storing the key as value in the pdc) ? Also should be static

Reply:
> It is considered good practice to reuse the `NamespacedKey`'s. Constructing them from a static context is ideal for this.

Is this better?

Reply:
Yes it's better